### DISCUSSION TITLE
Welcome to aspect-injector Discussions!

### DISCUSSION THREAD

**Discussion by pamidur** [pamidur/aspect-injector#139] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by williamamincey** [comment]
Howdy - would like to use AspectInjector to log with Serilog. I have a .NET core app that injects the ILogger at startup. How would I get that object into the Aspect? Would it need to be registered using Dependency Injection? Thanks!

**Reply by pamidur** [reply]
Hi @williamamincey , you can take a look at my comment here https://github.com/pamidur/aspect-injector/issues/128#issuecomment-644788660 . The end of the comment contains full example of usage with DI frameworks. Hope it helps !

I should probably add it as an example to the docs

**Reply by pamidur** [reply]
#166

**Comment by GabrielEngSist** [comment]
@pamidur Do you plan to upgrade dotnet framework version to 6.0?

**Reply by pamidur** [reply]
Hi @GabrielEngSist , I have just checked, version 2.6.1 works fine with net6. AspectInjector itself targets netstandard2.0, so it should be compatible with any framework that supports netstandar2.0.
Do you have any case in particular in which AspectInjector fails with net6?

**Reply by GabrielEngSist** [reply]
Man ! U R Fast A F!
I really don't. I'll gonna stress my tests here, and finding something, i'll let you know.
Thanks man ! (:
